.github/PULL_REQUEST_TEMPLATE/module_update.md

```diff
@@ -0,0 +1,64 @@
+<!--
+This project's monthly turnover might well be beyond 1500 PRs. Since we are community driven,
+let's do tit for tat: submitters are kindly ask to pick an open pull request for review.
```

Comment:
`are kindly asked`?

(not a native speaker though :sweat_smile: )

Reply:
"seien freundlick gebeten"

Reply:
I think @Ma27 was highlighting that `ask` is the wrong tense, and should be `asked`.

Reply:
You are right, my elevtive perception, sorry.

Reply:
Ohh, sorry, should've made that more obvious!